_std/defines/minimap.dm

```diff
@@ -1,6 +1,7 @@
 // Determines the minimap types the icon/area should be displayed on.
-#define MAP_DEFAULT		(1<<0)
-#define MAP_AI			(1<<1)
+#define MAP_ALL			(~0)
```

Comment:
this makes sense to me but maybe not people experienced with bitflags, either comment what this is doing or make a define that explains this (like BITS_ALL idk??)